### PR TITLE
#158487964/Users should see a landing page with effective UI/UX

### PR DESCRIPTION
**Pull request by phemonick**
#### What does this PR do?
convert mockup to html , css and angularjs
#### Description of Task to be completed?
Users should be able to see an overview of what the game is about, how it's played and how it can be used to donate to charity and also see call to action buttons that lead them to signup or signin
#### How should this be manually tested?
N/A
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#158487964 Users should see a landing page with effective UI/UX
#### Screenshots (if appropriate)
screen shot 2018-07-09 at 6 55 03 pm

#### Questions:

### CODE REVIEW

**Review by Stylll — COMMENTED** [review]
Nice work. However, I noticed there are no screenshots of the landing page in your PR request. You also have 3 console logs in the landingPage.js file. You should remove the console log messages and add a screenshot to your PR request.

**Review by ozimos — COMMENTED** [review]
Good work,
However, your PR title does not follow our PR Naming convention. Please rename the PR.

File: client/js/landingPage.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+window.addEventListener('load', () => {
+  let elems = document.querySelectorAll('.slider');
```
[the remark this answers is not in the document]
Comment (VictoriaOffoma) [reply client/js/landingPage.js]: Is there a reason why we are using _let_ and not _const_  for the _elems_ variable?

File: client/js/landingPage.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+  let instances = M.Slider.init(elems, {
```
[the remark this answers is not in the document]
Comment (VictoriaOffoma) [reply client/js/landingPage.js]: Is there a reason why we are using _let_ and not _const_ for the _instances_ variable?

File: client/css/landingpage.scss [hunk 3]
```diff
@@ -0,0 +1,363 @@
...
+  }
+  footer {
+    height: 80px;
+    color: #ffffff;
+    text-align: center;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-size: 16px;
+    background: #006989;
```
[the remark this answers is not in the document]
Comment (VictoriaOffoma) [reply client/css/landingpage.scss]: do we have our colours declared in variables?

**Pull request merged**